application/account-management/WebApp/shared/translations/locale/nl-NL.po

```diff
@@ -0,0 +1,281 @@
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: nl-NL\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: @lingui/cli\n"
+
+msgid "Account"
+msgstr "Account"
+
+msgid "Account settings"
+msgstr "Accountinstellingen"
+
+msgid "Account Settings"
+msgstr "Accountinstellingen"
+
+msgid "Actions"
+msgstr "Acties"
+
+msgid "Active Users"
+msgstr "Actieve gebruikers"
+
+msgid "Active users in the past 30 days"
+msgstr "Actieve gebruikers in de afgelopen 30 dagen"
+
+msgid "Add more in the Users menu"
+msgstr "Meer toevoegen via het gebruikersmenu"
+
+msgid "Added"
+msgstr "Toegevoegd"
+
+msgid "All Users"
+msgstr "Alle gebruikers"
+
+msgid "An error occurred while processing your request. {0}"
+msgstr "Er is een fout opgetreden bij het verwerken van uw verzoek. {0}"
+
+msgid "By continuing, you accept our policies"
+msgstr "Door verder te gaan, accepteer je onze voorwaarden"
+
+msgid "Can't find your code? Check your spam folder."
+msgstr "Kun je je code niet vinden? Controleer je spammap."
+
+msgid "Cancel"
+msgstr "Annuleren"
+
+msgid "Continue"
+msgstr "Doorgaan"
+
+msgid "Create your account"
+msgstr "Maak je account aan"
+
+msgid "Danger zone"
+msgstr "Gevaarzone"
+
+msgid "Delete"
+msgstr "Verwijderen"
+
+msgid "Delete Account"
+msgstr "Account verwijderen"
+
+msgid "Deleting the account and all associated data. This action cannot be undone, so please proceed with caution."
+msgstr "Het account en alle bijbehorende gegevens worden verwijderd. Deze actie kan niet ongedaan worden gemaakt, dus ga voorzichtig te werk."
+
+msgid "Didn't receive the code? Resend"
+msgstr "Code niet ontvangen? Verstuur opnieuw"
+
+msgid "Do you already have an account?"
+msgstr "Heb je al een account?"
+
+msgid "Domain"
+msgstr "Domein"
+
+msgid "Don't have an account? <0>Create one</0>"
+msgstr "Heb je geen account? <0>Maak er één aan</0>"
+
+msgid "E.g., CompanyX"
+msgstr "Bijv. BedrijfX"
+
+msgid "E.g., Marketing Manager"
+msgstr "Bijv. Marketing Manager"
+
+msgid "E.g., Olivia"
+msgstr "Bijv. Olivia"
+
+msgid "E.g., Rhye"
+msgstr "Bijv. Rhye"
+
+msgid "Edit profile"
+msgstr "Profiel bewerken"
+
+msgid "Email"
+msgstr "E-mail"
+
+msgid "Enter your email below to log in"
+msgstr "Voer je e-mailadres hieronder in om in te loggen"
+
+msgid "Enter your verification code"
+msgstr "Voer je verificatiecode in"
+
+msgid "Error: Something went wrong!"
+msgstr "Fout: Er is iets misgegaan!"
+
+msgid "Error: Verification code has expired"
+msgstr "Fout: Verificatiecode is verlopen"
+
+msgid "Europe"
+msgstr "Europa"
+
+msgid "Fetching data..."
+msgstr "Gegevens ophalen..."
+
+msgid "Filters"
+msgstr "Filters"
+
+msgid "First name"
+msgstr "Voornaam"
+
+msgid "Help"
+msgstr "Help"
+
+msgid "Here's your overview of what's happening."
+msgstr "Hier is je overzicht van wat er gebeurt."
+
+msgid "Hi! Welcome back"
+msgstr "Hallo! Welkom terug"
+
+msgid "Home"
+msgstr "Home"
```

Comment:
I couldn't find any example Dutch websites with a dedicated Home page (as least not with that name) But I think Home is indeed more applicable instead of Thuis.